Notices.md

```diff
@@ -0,0 +1,58 @@
+# Notices
+
+## Code of Conduct
+
+Contact for Code of Conduct issues or inquires:
+
+* Christopher Atkins @ MISO
+* Tory McKeag @ GE Vernova
```

Comment:
@getorymckeag We could consider asking LFE if they could field these, but for now, it's us.